modules/nw-ptp-operator-metrics-reference.adoc

```diff
@@ -82,27 +69,48 @@ a|Returns values for `HoldOverTimeout`, `MaxOffsetThreshold`, and `MinOffsetThre
 * `maxOffsetThreshold` and `minOffsetThreshold` are offset values in nanoseconds that compare against the values for `CLOCK_REALTIME` (`phc2sys`) or master offset (`ptp4l`) values that you configure in the `PtpConfig` CR for the NIC.
 |`{node="compute-1.example.com", profile="grandmaster", threshold="HoldOverTimeout"} 5`
 
-|`openshift_ptp_pps_status`
-|Returns the current status of the NIC 1PPS connection.
-You use the 1PPS connection to synchronize timing between connected NICs.
-Possible values are 0 (`UNAVAILABLE`) and 1 (`AVAILABLE`).
-|`{from="dpll",iface="ens2fx",node="compute-1.example.com",process="dpll"} 1`
+|====
+
+[discrete]
+== PTP fast event metrics only when T-GM is enabled
+
+The following table describes the PTP fast event metrics that are available only when PTP grandmaster clock (T-GM) is enabled.
+
+.PTP fast event metrics when T-GM is enabled
+[cols="1,4,3", options="header"]
+|====
+|Metric
+|Description
+|Example
+
+|`openshift_ptp_frequency_status`
+|Returns the current status of the digital phase-locked loop (DPLL) frequency for the NIC.
+Possible values are -1 (`UNKNOWN`), 0 (`INVALID`), 1 (`FREERUN`), 2 (`LOCKED`), 3 (`LOCKED_HO_ACQ`), or 4 (`HOLDOVER`).
```

Comment:
Feels like the number values should also be marked up (throughout the PR, in similar cell entries), but I think that might be outside of the scope of this PR, since it would make things inconsistent with other existing areas in this assembly. Plus, it would be a lot of markup in one area, considering the parentheticals, so maybe take this question with a grain of salt. 😅

Reply:
Hi @adellape 
I agree with you! I will create a separate PR to work on this update.
Thank you!